lib/Runtime/Language/JavascriptStackWalker.cpp

```diff
@@ -606,15 +698,15 @@ namespace Js
                 {
                     // Done walking inlined frames in a loop body, cache the native code address now
                     // in order to skip the loop body frame.
-                    this->SetCachedInternalFrameInfo(InternalFrameType_LoopBody, InternalFrameType_LoopBody);
+                    this->SetCachedInternalFrameInfo(InternalFrameType_LoopBody, InternalFrameType_LoopBody, true /*inlinedFramesOnStack*/);
                     isJavascriptFrame = false;
                 }
             }
 
             return true;
         }
 #endif
-
+        this->inlinedFramesOnStack = false;
```

Comment:
Which class member/local are you talking about? All the new members I have added don't have the 'is' prefix.

---
In reply to: [64985811](https://github.com/Microsoft/ChakraCore/pull/1052#discussion_r64985811) [](ancestors = 64985811)